examples/next13-typescript-static-pages/README.md

```diff
@@ -1,6 +1,17 @@
 This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
```

Comment:
```suggestion
# next-firebase-auth-edge static pages example

This is a [Next.js](https://nextjs.org/) project showcasing `next-firebase-auth-edge` library features.

Demo of this project can be previewed at [next-firebase-auth-edge-static-demo.vercel.app](https://next-firebase-auth-edge-static-demo.vercel.app/) 
```

Reply:
Thanks for creating a PR. I really appreciate it.  🙏 

Let's add this introduction, so that people don't automatically assume it's a boilerplate Next.js readme